File: Tugas coding modul 6/coding c/PRAK601-2310817120005-Noviana_Nur_Aisyah.c
#include <stdio.h>

void matriks(int baris, int kolom, int m[baris][kolom]) {
    for (int i = 1; i <= baris; i++) {
        for (int j = 1; j <= kolom; j++) {
            printf("%d ", m[i][j]);
        }
        printf("\n");
    }
}

int main() {
    int baris, kolom;

    scanf("%d %d", &baris, &kolom); //baris dan kolom

    int m[baris][kolom];
    for (int i = 1; i <= baris; i++) {
        for (int j = 1; j <= kolom; j++) {
            scanf("%d", &m[i][j]);
        }
    }

    printf("\n");
    matriks(baris, kolom, m);

    return 0;
}

File: Tugas coding modul 6/coding py/PRAK601-2310817120005-Noviana_Nur_Aisyah.py
def matriks(baris, kolom, m):
    for i in range(baris):
        for j in range(kolom):
            print(m[i][j], end=" ")
        print()

baris, kolom = map(int, input().split())  

m = []
for _ in range(baris):
    m.append([0] * kolom)

input_aja = list(map(int, input().split()))

matrix = 0
for i in range(baris):
    for j in range(kolom):
        m[i][j] = input_aja[matrix]
        matrix += 1

print()
matriks(baris, kolom, m)